pkg/controllers/status/crb_status_controller_test.go

```diff
@@ -100,6 +101,7 @@ func TestCRBStatusController_Reconcile(t *testing.T) {
 				ObjectMeta: metav1.ObjectMeta{
 					Name:              "binding",
 					Namespace:         "default",
+					Finalizers:        []string{"test"},
```

Comment:
the `Finalizers` field is required, otherwise the function `.WithObjects(binding).WithStatusSubresource(binding)` will failed with the error `refusing to create obj binding with metadata.deletionTimestamp but no finalizers`

Reply:
We can set it in the comment, which facilitates understanding during subsequent simulations.